.travis.yml

```diff
@@ -33,3 +33,6 @@ script:
   - make eunit
   - # ERL_LIBS="$HOME/proper" ERLC_OPTS='-DPROPER' make compile-test
   - # ERL_LIBS="$HOME/proper" make eunit
+
+after_success:
+  - make coverage-report
```

Comment:
It is cause a build fail on travis-ci if organisation dont use this service
https://travis-ci.org/dinkor/kazoo/jobs/170391627